src/index.js

```diff
@@ -106,7 +107,7 @@ export default ({
             const remoteModuleMap = JSON.parse(r.body);
 
             const { modules } = remoteModuleMap;
-            const oneAppDevStaticsAddress = `http://localhost:${process.env.HTTP_ONE_APP_DEV_CDN_PORT}/static`;
+            const oneAppDevStaticsAddress = useHost ? `http://${req.headers.host}/static` : `http://localhost:${process.env.HTTP_ONE_APP_DEV_CDN_PORT}/static`;
```

Comment:
continued from comment above -
```suggestion
            const oneAppDevStaticsAddress = `${hostAddress}/static`;
```
this change can also be applied below

Reply:
https://github.com/americanexpress/one-app-dev-cdn/pull/14/commits/1c5a17c9779e2264f2584d27afc1031b0fdfd53b